PeerioTechnologies/peerio-desktop
# fix: extract message text into component to preserve selection between renderings

#### Relevant info and issue/PR links

https://app.clubhouse.io/peerio/story/17120/desktop-when-selecting-message-text-selection-clears-if-new-read-receipt-appears

#### Testing instructions

1. Send a message to another account. Don't let the other account read it yet.
2. Select some part of the sent message text (after it was sent, because there's still an issue when selecting text when it was unsent).
2. In another account (in a different instance or on mobile), read the message.
3. In the original app, check that selection is still there after the read receipt appears.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
